profile/README.md

```diff
@@ -1,12 +1,36 @@
-## Hi there 👋
+## Welcome to the Autoware Foundation! 👋
+![Autoware_RViz](https://user-images.githubusercontent.com/63835446/158918717-58d6deaf-93fb-47f9-891d-e242b02cba7b.png)
 
-<!--
+The [Autoware Foundation](https://www.autoware.org/) manages the development of Autoware, the world's leading open-source software project for autonomous driving, and sponsors Autoware-based projects all over the world. Autoware was started in 2015 and is being used by over 100 companies on more than 30 vehicles in over 20 different countries worldwide. 
 
```

Comment:
```suggestion

[![Discord](https://img.shields.io/discord/953808765935816715?label=Join%20Autoware%20Discord&style=for-the-badge)](https://discord.gg/Q94UsPvReQ)

```

Reply:
Added this here with suggestion made in https://github.com/autowarefoundation/autoware-documentation/pull/58#pullrequestreview-913859438